lib/auth/Vault.ts

```diff
@@ -36,7 +36,7 @@ function vaultSignatureCb(
         return callback(err);
     }
 
-    const { email, ...userInfoWithoutEmail } = authInfo.message.body.userInfo;
+    const { ...userInfoWithoutEmail } = authInfo.message.body.userInfo;
 
     log.debug('received info from Vault', {
         ...authInfo,
```

Comment:
The goal of this was to exclude the email from the info we set for `userInfo` below, with the new change it's included again.

Reply:
I'm pretty sure the email was removed from the log for security reasons, you check this commit https://github.com/scality/Arsenal/commit/6374fc831d9416c285a1d96ca18f614533ee8e5d (the name of the variable also suggests it)

Reply:
We can try another approach if we want to avoid the linting issue but yes, we should not print PII in the logs, and the auth info has some (the email)

Reply:
this should do it ?

```suggestion
    const { email: _, ...userInfoWithoutEmail } = authInfo.message.body.userInfo;

    log.debug('received info from Vault', {
        ...authInfo,
```